wormhole-connect/src/utils/transferValidation.ts

```diff
@@ -107,6 +108,9 @@ export const validateAmount = (
     const b = Number.parseFloat(balance);
     if (numAmount > b) return 'Amount cannot exceed balance';
   }
+  if (maxAmount !== undefined && numAmount > maxAmount) {
+    return `At the moment max amount cannot exceed ${maxAmount}`;
```

Comment:
typo? "max amount cannot exceed" should just read "amount cannot exceed"

Reply:
changed